File: src/com/dafttech/wai/DesktopStarter.java
package com.dafttech.wai;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;

public class DesktopStarter {
	public static void main(String[] args) {
		new LwjglApplication(new TerraInfinita(), "TerraInfinita", 800, 600, false);
	}
}

File: terra/src/main/scala/com/dafttech/terra/DesktopStarter.scala
package com.dafttech.terra

import com.badlogic.gdx.backends.lwjgl.{LwjglApplication, LwjglApplicationConfiguration}

/**
  * Created by LolHens on 18.04.2015.
  */
object DesktopStarter {
  private def createConfig(title: String, width: Int, height: Int): LwjglApplicationConfiguration = {
    val config = new LwjglApplicationConfiguration()
    config.title = title
    config.width = width
    config.height = height
    config.vSyncEnabled = true
    //config.useGL30 = true
    config
  }

  def main(args: Array[String]): Unit = new LwjglApplication(TerraInfinita, createConfig("TerraInfinita", 1440, 900))
}
